styles/work-section.css

```diff
@@ -287,6 +289,154 @@
   color: white;
 }
 
+.project-pop-up {
+  display: none;
+  flex-direction: column;
+  align-content: center;
+  width: 85%;
+  position: fixed;
+  z-index: 5;
+  top: 4.5%;
+  margin-right: 1.5%;
+  margin-left: 1.5%;
+  height: 670px;
+  border: 0;
+  background-color: white;
+  left: 5%;
+  backdrop-filter: blur(10px);
+}
+
+.project-pop-up:active {
+  display: flex;
+}
+
+.project-popup-floating {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 85%;
+}
+
+.pop-up-title-div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 85%;
+}
+
+.popup title {
+  font-size: 20px;
+}
+
+.bar-x-popup {
+  display: block;
+  top: 100px;
+  width: 20px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all ease-in-out;
+  background-color: #67798e;
+  border-radius: 10px;
+  margin-right: 30px;
+  margin-top: 10px;
+}
+
+.hamburger-X-popup {
+  cursor: pointer;
+  margin-top: 15px;
+  margin-left: 20px;
+}
+
+.hamburger-X-popup .bar-x-popup:nth-child(1) {
+  transform: translateY(6px) rotate(45deg);
+}
```

Comment:
- [ ] Kindly make changes for the CSS part from the loom video attached [here](https://www.loom.com/share/7d926bce92c74d789cad68f04f2f61d6).
- [ ] Follow the figma design with font family, font size, spacing etc.